pkg/controller/utils/controllerwrapper.go

```diff
@@ -0,0 +1,68 @@
+package utils
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/go-logr/logr"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var controllerMaxReconciles map[string]int = map[string]int{}
+
+func NewControllerWithMaxReconciles(log logr.Logger, controllerName string, mgr manager.Manager, r reconcile.Reconciler) (controller.Controller, error) {
+	maxConcurrentReconciles, err := getControllerMaxReconciles(controllerName)
+	if err != nil {
+		log.Error(err, "")
+	}
+	return controller.New("account-controller", mgr, controller.Options{Reconciler: r, MaxConcurrentReconciles: maxConcurrentReconciles})
```

Comment:
```suggestion
	return controller.New(fmt.Sprintf("%s-controller", controllerName), mgr, controller.Options{Reconciler: r, MaxConcurrentReconciles: maxConcurrentReconciles})
```

Reply:
ohh deerrrppppppp.

Something something mondays.

Thanks for pointing that out :)